modules/ROOT/pages/layers.adoc

```diff
@@ -15,12 +15,11 @@ Map a node to a layer in the application's properties file, or using the UI in t
 
 === Set a Layer with Properties
 
-To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>` in the application's properties file. Note that this is an application property, not a server property.
-
-When you use this property, nodes automatically appear in the specified layer. If the layer doesn't exist, it is created.
+To automate adding layer(s) to a node, use the property `anypoint.platform.visualizer.layer=<name>` in the application’s properties file, or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property. When you use this property, nodes automatically appear in the specified layer. If the layer doesn't exist, it is created.
```

Comment:
GLOBALLY replace anything with "(s)" with the more likely of the two: singular or plural. Plural is sometimes better because it is inclusive of the two, but, well, that's a subjective interpretation.
Or you could use "layer or layers"....

Reply:
I just chose one at random:
```suggestion
You can automate adding layers to a node by using `anypoint.platform.visualizer.layer=<name>`, either in the application’s properties file or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property. When you use this property, nodes automatically appear in the specified layer and, if that layer doesn't exist, it is automatically created.
```

Reply:
Wait. The beginning specifies adding layers to a node, but later says that nodes appear in the layer? What is being added to what?

Also, I wonder if we can incorporate the Note info into this, and if we really need to repeat automating creation?
Is the following technically correct?

```suggestion
You can automate adding layers to a node by using `anypoint.platform.visualizer.layer=<name>`, either in the application’s properties file (non-Mule 4 deployments) or at deployment time through the Runtime Manager UI or API (required for Mule 4). Note that this is an application property, not a server property. 
```

Reply:
```suggestion
You can automate adding nodes to a layer by using `anypoint.platform.visualizer.layer=<name>`, either in the application’s properties file (non-Mule 4 deployments) or at deployment time through the Runtime Manager UI or API (required for Mule 4). Note that this is an application property, not a server property. 
```